src/admin-views/editor/panel/ticket.php

```diff
@@ -26,7 +26,7 @@
  * @var string                             $ticket_type                      The type of Ticket the form is for.
  */
 
-$ticket_type = $ticket_type ?: 'default';
+$ticket_type = $ticket_type ?? 'default';
```

Comment:
Why this change?
`?:` means "if the ticket_type is empty use default", while `??` means "if the ticket_type is NULL use default". We want to use "default" when the ticket type is an empty string.

Reply:
When the template is loaded for New Events/posts, there are no tickets associated with that and we don't pass any:
https://github.com/the-events-calendar/event-tickets/blob/0b7bb6078d32708fa53ec8d53cd0c790b1a6c1f0/src/Tribe/Metabox.php#L109

We can either pass a `ticket_type` var on the linked location or check for `null` to avoid the warning on the new post/event pages.